src/Word.elm

```diff
@@ -308,7 +313,7 @@ add x y =
                     xl + yl
 
                 zh =
-                    xh + yh + rem32 zl
+                    xh + yh + carry32 xl yl
```

Comment:
Another issue on windows was the way we calculate the 32 bit carry value from addition. `carry32` implements a slightly more expensive algorithm that works on all platforms